### DISCUSSION TITLE
Edit icon on hover over

### DISCUSSION THREAD

**Discussion by rishachilles** [ghiscoding/slickgrid-universal#1034] [Q&A] [answered]
When we hover over a editable cell we can see a pencil icon in examples how do we achieve that in salesforce?

**Comment by ghiscoding** [comment, marked as answer]
The exact same as the Example, create a Custom Formatter and use it. Formatters are returning string in most cases, just create a Formatter with an href and css class to do this effect or just copy the one from the Example.
